lib/core/src/server/build-static.js

```diff
@@ -72,42 +40,102 @@ export async function buildStaticStandalone(options) {
     });
   }
 
-  // compile all resources with webpack and write them to the disk.
-  return new Promise((resolve, reject) => {
-    const previewStartTime = process.hrtime();
-
-    const webpackCb = (err, stats) => {
-      if (err || stats.hasErrors()) {
-        logger.error('Failed to build the storybook');
-        // eslint-disable-next-line no-unused-expressions
-        err && logger.error(err.message);
-        // eslint-disable-next-line no-unused-expressions
-        stats && stats.hasErrors() && stats.toJson().errors.forEach(e => logger.error(e));
-        process.exitCode = 1;
-        return reject(err);
+  logger.info('copy dll..');
+  shelljs.cp(dllPath, path.join(outputDir, 'sb_dll'));
+
+  let startTime;
+
+  return new Promise(res => {
+    logger.info('building manager..');
+    startTime = process.hrtime();
+
+    res();
+  })
+    .then(() =>
+      loadManagerConfig({
+        configType,
+        outputDir,
+        configDir,
+        corePresets: [require.resolve('./manager/manager-preset.js')],
+      })
+    )
+    .then(
+      managerConfig =>
+        new Promise((resolve, reject) => {
+          webpack(managerConfig).run((err, stats) => {
+            if (err || !stats || stats.hasErrors()) {
+              logger.error('Failed to build the manager');
+              // eslint-disable-next-line no-unused-expressions
+              err && logger.error(err.message);
+              // eslint-disable-next-line no-unused-expressions
+              stats && stats.hasErrors() && stats.toJson().errors.forEach(e => logger.error(e));
+              process.exitCode = 1;
+
+              reject(stats);
+              return;
+            }
+
+            resolve(stats);
+          });
+        })
+    )
+    .then(stats => {
+      logger.trace({ message: 'manager built', time: process.hrtime(startTime) });
+      stats.toJson().warnings.forEach(e => logger.warn(e));
+    })
+    .then(() => {
+      startTime = process.hrtime();
+      logger.info('building preview..');
+
+      return loadConfig({
+        configType,
+        outputDir,
+        packageJson,
+        corePresets: [require.resolve('./preview/preview-preset.js')],
+        overridePresets: [require.resolve('./preview/custom-webpack-preset.js')],
+        ...options,
+      });
+    })
+    .then(
+      config =>
+        new Promise((resolve, reject) => {
+          webpack(config).run((err, stats) => {
```

Comment:
in case we have a "watch" parameter passed, we should run `webpack(config).watch()` for manager. Otherwise it's a breaking change =(

Reply:
I couldn't find any usage of this though, how would this ever be used by users?

Reply:
Issue: #2762
PR: #2866

Reply:
I see, I didn't know about this.